src/rashdf/plan.py

```diff
@@ -1,9 +1,79 @@
 from .geom import RasGeomHdf
-
+from .base import RasHdf
+from typing import Dict
 from geopandas import GeoDataFrame
 
 
-class RasPlanHdf(RasGeomHdf):
+class RasPlanHdf(RasHdf):
+
+    def __init__(self, name: str):
+        super().__init__(name)
+        self.plan_info_path = "Plan Data/Plan Information"
+        self.plan_params_path = "Plan Data/Plan Parameters"
+        self.meteorology_precip_path = "Event Conditions/Meteorology/Precipitation"
+        self.results_unsteady_path = "Results/Unsteady"
+        self.results_summary_path = "Results/Unsteady/Summary"
+        self.volume_accounting_path = "Results/Unsteady/Summary/Volume Accounting"
```

Comment:
Similar to above, these constants can be class properties. Since this is ultimately a subclass of `h5py.File`, it'll also be important to have `**kwargs` in the `__init__` method in order to pass any optional arguments down to that level.

```python
class RasPlanHdf(RasHdf):
    PLAN_INFO = "Plan Data/Plan Information"
    PLAN_PARAMS = "Plan Data/Plan Parameters"
    PRECIP = "Event Conditions/Meteorology/Precipitation"
    RESULTS_UNSTEADY = "Results/Unsteady"
    RESULTS_SUMMARY = "Results/Unsteady/Summary"
    VOLUME_ACCOUNTING = "Results/Unsteady/Summary/Volume Accounting"

    def __init__(self, name: str, **kwargs):
        super().__init__(name, **kwargs)
```